openwisp_users/multitenancy.py

```diff
@@ -134,3 +136,52 @@ class MultitenantRelatedOrgFilter(MultitenantOrgFilter):
     """
 
     multitenant_lookup = 'organization__in'
+
+
+class BaseOrganizationPermission(BasePermission):
+    def get_organization(self, view, obj):
+        organization_field = getattr(view, 'organization_field', 'organization')
+        return getattr(obj, organization_field, None)
```

Comment:
what if the organization field is on a parent object?

Reply:
https://github.com/atb00ker/openwisp-users/tree/issues/150-org-member-validator#organization_field